server/data/apps.json

```diff
@@ -3600,5 +3600,70 @@
         "elementURL": "https://www.podigee.com/en/podcast-transcriptions"
       }
     ]
+  },
+  {
+    "appName": "Oncetold",
+    "appType": [
+      "hosting"
+    ],
+    "appUrl": "https://oncetold.us",
+    "appIconUrl": "oncetold.png",
+    "platforms": [
+      "Web"
+    ],
+    "supportedElements": [
+      {
+        "elementName": "GUID",
+        "elementURL": "https://www.podhome.fm/docs/podcasting-2-0"
```

Comment:
Shouldn't the elementURLs point to the info on your platform?

Reply:
Yup! I need to edit it. Sorry